src/connectors/index.ts

```diff
@@ -140,11 +140,11 @@ export const onboard = init({
     en: {
```

Comment:
why do we need all of this at all? can't we just use the default texts?

Reply:
In short, this is their default text.
https://github.com/blocknative/web3-onboard/blob/v2-web3-onboard/packages/core/src/i18n/en.json

Initially, I put it in to start work on localization. Which I don't know how to handle yet, but which is easier discussed in our call tomorrow I guess?

But now also, feedback came in from the designer who requested some of the texts to be updated.
And you cant customize/update just 1, you have to add them all.